modules/core/src/experimental/utils/gpu-grid-aggregator.js

```diff
@@ -372,7 +372,8 @@ export default class GPUGridAggregator {
         gridSize,
         uProjectionMatrix: gridTransformMatrix,
         uProjectionMatrixFP64,
-        projectPoints
+        projectPoints,
+        ONE: 1.0
```

Comment:
This uniform should actually be set by `fp64` module. I will fix this separately on `master` (this is for 6.0-release branch). I will add `getUniform` method for `fp64` module in luma.gl and remove this uniform setting from `Layer` class in deck.gl